components/base/myUser.tsx

```diff
@@ -0,0 +1,23 @@
+import { GlobalContext } from "@/contexts/globalContext";
+import { useContext, useState } from "react";
+
+export const MyUserComponent = () => {
```

Comment:
Lets not call components "component"
also every component is "my" component
so how about "user form" or "user changer"?